.rubocop.yml

```diff
@@ -0,0 +1,21 @@
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  Exclude:
+    - "vendor/**/*"
+    - "tests/integration/**/roles/*"
+    # ignore these until rubocop is incorportaed into qansible
+    - Rakefile
+    - Gemfile
+Style/StringLiterals:
+  # needs agreement
```

Comment:
will fix. we should document how we make our own policy. would you leave your comment in the issue?